src/datadeps.jl

```diff
@@ -36,13 +36,12 @@ const ghcnd_data = "ghcnd_all.tar.gz"
 const ghcnd_data_dir = "ghcnd_all"
 
 """
-    uncompress_all_data(fname="ghcnd_all.tar.gz")
+    uncompress_all_data(fname=$ghcnd_data)
 
-`download_all_data` obtained a `tar.gz`. This function uncompresses it. Assumes a working
-installation of tar.
+`download_all_data` obtained a `tar.gz`. This function uncompresses it.
 """
-function uncompress_all_data(fname="ghcnd_all.tar.gz")
-    run(`tar xzvf $(ghcnd_data * "tar.gz")`)
+function uncompress_all_data(fname=ghcnd_data)
+    unpack(fname)
 end
```

Comment:
I think this code was just wrong before.

Reply:
This is what I get for not writing tests. This entire function can be deleted.